tools/getkeys/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Was this added to make our eslint config happy? This package doesn't contain any typescript code, so ideally we shouldn't have to add this. That said, I wouldn't be surprised if the current configuration doesn't handle this nicely. 

@tylerbutler do we have anything tracking this issue already? Or should we make one.

Reply:
> 

It probably because I set `legacy-peer-deps` true on npm config. Removing it and rebuild...

Reply:
Seems like it's still requested to use lint in this package. Troubleshooting the root cause...

Reply:
If this is just to make eslint happy, we should probably add a comment in here explaining why it is needed. We can then remove it later when we fix our config to not need this for js-only packages.

Reply:
> If this is just to make eslint happy, we should probably add a comment in here explaining why it is needed. We can then remove it later when we fix our config to not need this for js-only packages.

Got it. Adding comments here...